ang/civicase/case/card/directives/case-activity-count.directive.html

```diff
@@ -43,12 +43,10 @@
   <span>Other: </span>
   <a class="civicase__case-card__activity-count"
     ng-if="data.category_count.other.incomplete"
-    ng-href="{{getActivityFeedUrl({ caseId: data.id, category: otherCategories, statusType: 'incomplete' })}}"
   > {{ data.category_count.other.incomplete }} </a>
```

Comment:
this is not a link anymore, should it be an anchor tag? why not use a span? (please check below as well)